Patterns/Portuguese/Portuguese-DateTime.yaml

```diff
@@ -311,8 +311,8 @@ MidTimeRegex: !nestedRegex
   references: [ MidnightRegex, MidmorningRegex, MidafternoonRegex, MiddayRegex, MidEarlyMorning ]
 AtRegex: !nestedRegex
   # "as quatro" "às 3"
-  def: \b(((?<=\b([aà]s?)\s+)({WrittenTimeRegex}|{HourNumRegex}|{BaseDateTime.HourRegex})(\s+horas?|\s*h\b)?|(?<=\b(s(er)?[aã]o|v[aã]o\s+ser|^[eé]h?)\s+|^\s*)({WrittenTimeRegex}|{HourNumRegex}|{BaseDateTime.HourRegex})(\s+horas?|\s*h\b))(\s+{OclockRegex})?|{MidTimeRegex})\b
-  references: [ HourNumRegex, BaseDateTime.HourRegex, WrittenTimeRegex, OclockRegex, MidTimeRegex ]
+  def: \b(((?<=\b([aà]s?)\s+)({WrittenTimeRegex}|{HourNumRegex}|{BaseDateTime.HourRegex}(\s+e\s+{BaseDateTime.MinuteRegex})?)(\s+horas?|\s*h\b)?|(?<=\b(s(er)?[aã]o|v[aã]o\s+ser|^[eé]h?)\s+|^\s*)({WrittenTimeRegex}|{HourNumRegex}|{BaseDateTime.HourRegex})(\s+horas?|\s*h\b))(\s+{OclockRegex})?|{MidTimeRegex})\b
+  references: [ HourNumRegex, BaseDateTime.HourRegex, BaseDateTime.MinuteRegex, WrittenTimeRegex, OclockRegex, MidTimeRegex ]
 ConnectNumRegex: !nestedRegex
```

Comment:
I believe we need to add at least some abiguity filter for cases like "entre 3 e 10" ("betweeb 3 and 10"). Also, does "reunião às cinco e quarenta" also works? I.e., same case with written numbers?

Reply:
As for your comment in the issue, the pattern is extracted only if preceded by the preposition "às", so cases like "entre 3 e 10" are already excluded.
Patterns with written numbers should already be supported (they are included in WrittenTimeRegex) but I just noticed  they are not parsed correctly, so I will try to fix that.

Reply:
Cool. Can you add a spec case with "entre" to show it works as expected (no extraction). 
Also, does a complex case like "entre as 5 e as 7 da manha" work as a timerange? If not, let's open a low-priority issue for it.

Reply:
fixed parsing of written numbers and added specs.